tests/kuttl/tests/dataplane-create-test/00-assert.yaml

```diff
@@ -9,12 +9,12 @@ spec:
 status:
   conditions:
   - message: Deployment not started
-    reason: NotRequested
+    reason: Requested
```

Comment:
This doesn't look right. A deployment hasn't been requested in this test, so I don't think this should change.